cms/envs/common.py

```diff
@@ -927,6 +927,7 @@
 
     # This must be last so that it runs first in the process_response chain
     'openedx.core.djangoapps.site_configuration.middleware.SessionCookieDomainOverrideMiddleware',
+
```

Comment:
nit: let's remove this newline to keep the diff minimal.

Reply:
Ack. Will do.